content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,143 @@
-   required) and paste the token, then click on next.
+#. Upon connecting to the :abbr:`IoT (Internet of Things)` WiFi a browser will automatically
+   redirect to the :abbr:`IoT (Internet of Things)` Box Homepage. Give a name to the
+   :abbr:`IoT (Internet of Things)` Box, paste the previously copied token into the
+   :guilabel:`Server Token` field, and then click on :guilabel:`Next`.
 
-   .. image:: connect/connect07.png
+   .. image:: connect/server-token.png
+      :align: center
+      :alt: Enter the server token into the IoT box.
 
-   .. tip::
-      If you are on Runbot, do not forget to add the -all or -base in the
-      token (e.g. this token
-      **http://375228-saas-11-5-iot-f3f920.runbot16.odoo.com\|4957098401**
-      should become
-      **http://375228-saas-11-5-iot-f3f920-all.runbot16.odoo.com\|4957098401**).
+   .. note::
+      If the :abbr:`IoT (Internet of Things)` Box WiFi connection wizard doesn't start navigate to
+      the :abbr:`IoT (Internet of Things)` Box admin console page by entering the  
 
-#. Choose the WiFi network you want to connect with (enter the password
-   if there is one) and click on Submit. Wait a few seconds before being
-   redirected to your database.
+#. Choose the WiFi network that the :abbr:`IoT (Internet of Things)` box will connect with (enter
+   the password if there is one) and click on :guilabel:`Connect`. Wait a few seconds before being
+   redirected to the database. The computer may need to be manually connected back to WiFi.
 
-   .. image:: connect/connect08.png
+   .. image:: connect/configure-wifi-network-iot.png
+      :align: center
+      :alt: Configuring the WiFi for the IoT box.
 
-You should now see the IoT Box.
+The :abbr:`IoT (Internet of Things)` box should appear in the :menuselection:`IoT` module of the
+Odoo database.
 
-.. image:: connect/connect09.png
+.. image:: connect/iot-box-connected.png
    :align: center
+   :alt: The IoT box has been successfully configured on the Odoo database.
+
+.. important::
+   A manual reset of the :abbr:`IoT (Internet of Things)` box may be necessary upon successfully
+   connecting the :abbr:`IoT (Internet of Things)` box via WiFi for the :abbr:`IoT
+   Internet of Things)` box to appear in the :menuselection:`IoT` module of the Odoo database.
+
+.. _iot_connect/token:
+
+Manually connecting the IoT via the token
+=========================================
+
+A manually connection of the :abbr:`IoT (Internet of Things)` box to the :menuselection:`IoT
+(Internet of Things)` module can be made with the :guilabel:`token`. The :guilabel:`token` can be
+found after clicking on :guilabel:`Connect` in the :menuselection:`IoT` module. The
+:guilabel:`token` will be inputted into the :abbr:`IoT (Internet of Things)` box admin console.
+
+#. Access the :abbr:`IoT (Internet of Things)` box admin console by entering the :abbr:`IP
+   (Internet Protocol)` of the :abbr:`IoT (Internet of Things)` box into a browser window.
```

Comment:
... by entering the :abbr:`IP (Internet Protocol)` of the ...
Technically speacking, I think we should rather say "IP address". In practice we use juste "IP" as a shortcut/malapropism but I think it is ambiguous in this case as it can be understood as the "protocol" itself

Reply:
I've made this change, thanks @lse-odoo